### PR TITLE
Bump SQLite version

### PR DESCRIPTION
**Pull request by quinnj**

### CODE REVIEW

**Comment by tkelman** [comment]
Is this a rewrite since the last tag 7 months ago? If so, then please use a new minor version number.

**Comment by tkelman** [comment]
Nevermind, looks like the rewrite was for version 0.3. Many of the git tags are missing so it's not easy to cross-reference what changes were made when. Please recreate the git tags to match the metadata tags if you no longer have them locally.

**Comment by tkelman** [comment]
This does however fail its Travis tests, even on release. Needs to be tighter about a dependency version, or is something just broken?

**Comment by quinnj** [comment]
The tests are failing because it's waiting on the CSV version bump.

**Comment by tkelman** [comment]
Then that needs to be listed explicitly in REQUIRE.

**Comment by quinnj** [comment]
Looks like travis is happy here now too: https://travis-ci.org/JuliaDB/SQLite.jl

File: SQLite/versions/0.3.2/requires [hunk 1]
```diff
@@ -1,6 +1,6 @@
 julia 0.4.1
 BinDeps
-CSV
+CSV 0.0.8
```
Comment (tkelman) [inline SQLite/versions/0.3.2/requires]: do this in the package too https://github.com/JuliaDB/SQLite.jl/blob/master/REQUIRE

File: SQLite/versions/0.3.2/requires [hunk 2]
```diff
@@ -0,0 +1,9 @@
+julia 0.4.1
+BinDeps
+CSV
+DataStreams
+DataFrames
```
Comment (tkelman) [inline SQLite/versions/0.3.2/requires]: you're doing something that is causing this not to match https://github.com/JuliaDB/SQLite.jl/blob/1123351beaba878eaf45cbc94a0ad1149273b1ef/REQUIRE - are you not even using `Pkg[Dev].tag`?

File: SQLite/versions/0.3.2/requires [hunk 3]
```diff
@@ -0,0 +1,9 @@
+julia 0.4.1
+BinDeps
+CSV 0.0.8
```
Comment (quinnj) [inline SQLite/versions/0.3.2/requires]: I'm showing the right dependency here, @tkelman

Comment (tkelman) [inline SQLite/versions/0.3.2/requires]: yeah this is good now, but whatever you did that resulted in metadata requires not matching the package requires, avoid making it a habit

**Pull request merged**